conf/npm/README.md

```diff
@@ -0,0 +1,4 @@
+This is a javascript library that helps you build search experiences on top of
```

Comment:
could you clarify what you mean by "language specific to github" nothing really stood out to me but I'm also illiterate

Reply:
sure,
from @tmeyer2115 :
```
The Answers Javascript API Library does not need to be installed locally. Instead, it can be used with script tags on a webpage. The instructions below explain how to do this; they will walk you through adding the Answers stylesheet, JS library, and an intialization script to an HTML page. After doing this, you can view your page in the browser.
```

where it mentions not being installed locally, though I could see the case where we are saying "it does not _need_ to be installed locally, you can also do this instead"

also got feedback that the README as it exists currently may be too large for NPM